assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -123,6 +123,67 @@ describe( 'modules/analytics properties', () => {
 				expect( properties ).toEqual( undefined );
 			} );
 		} );
+		describe( 'selectProperty', () => {
```

Comment:
Please keep a blank line between `describe` and `it` blocks 👍 
```suggestion

		describe( 'selectProperty', () => {
```